docs/developer-guide/maps-configuration.md

```diff
@@ -78,6 +79,18 @@ i.e.
     "center": {"x": 1000000.000000, "y": 5528000.000000, "crs": "EPSG:900913"},
     "zoom": 15,
     "visualizationMode": "2D",
+    "viewerOptions": {
+      "cameraPosition": {
+        "longitude": 0,
+        "latitude": 0,
```

Comment:
in this case, center is ignored from now on ?

Make you sure that this is still working.

https://dev-mapstore.geosolutionsgroup.com/mapstore/#/viewer/42750?center=13.365138002695128,52.44873896136653&zoom=18.49618392498629&heading=0.04401620072489898&pitch=-0.2864607523022795&roll=0.00005618807344109911